src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Props } from '../../types/Props';
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <>
+      <h1 className="title">
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                className={classNames({
+                  'is-active': tab.id === selectedTab.id,
+                })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTabId !== tab.id) {
+                      onTabSelected(tab);
+                    }
```

Comment:
No need in writing `if` because `selectedTabId` won't change:
```suggestion
                      onTabSelected(tab);
```